src/xtgeo/grid3d/grid.py

```diff
@@ -1792,17 +1902,20 @@ def get_cell_volume(
 
         .. versionadded:: 2.13 (as experimental)
         """
-        vol = _grid_etc1.get_cell_volume(
+        return _grid_etc1.get_cell_volume(
             self,
             ijk=ijk,
             activeonly=activeonly,
             zerobased=zerobased,
             precision=precision,
         )
 
-        return vol
-
-    def get_bulk_volume(self, name="bulkvol", asmasked=True, precision=2):
+    def get_bulk_volume(
+        self,
+        name: str = "bulkvol",
+        asmasked: bool = True,
+        precision: int = 2,
+    ) -> GridProperty:
```

Comment:
The doc string states that only 1, 2, and 4 are supported, so maybe a `Literal[1, 2, 4]` is warranted here. It will raise an exception further in:

https://github.com/equinor/xtgeo/blob/3a3507c4dcae7d03380294de57f62382953a63cb/src/xtgeo/grid3d/_grid_etc1.py#L211-L212